src/sentry/migrations/0262_drop_sentry_app_from_integrationfeature.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.2.24 on 2021-12-10 22:13
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0261_prepare_remove_sentry_app_column"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[],
+            database_operations=[
+                migrations.RunSQL(
+                    sql="ALTER TABLE sentry_integrationfeature DROP COLUMN sentry_app_id",
+                    reverse_sql="ALTER TABLE sentry_integrationfeature ADD COLUMN sentry_app_id int NULL",
```

Comment:
If the migration rolls back then we won't be able to populate old `sentry_app_id`'s right?

Reply:
Yeah I wasn't actually sure what to put here, cause they'll be lost so I just went with nothing 🤷

Reply:
In the past, I renamed a column to something like `sentry_app_id_deprecated` and on the roll back would just rename it back.

Reply:
The rollback is just a courtesy for dev environments, it is never used in production. We typically do the bare minimum here to allow migrations to work forwards/backwards.

Reply:
then LGTM :)